docs/design/semantic_slots.md

```diff
@@ -64,7 +64,7 @@ In the table below, you can see all eight palette colors defined for the bodyTe
 
 ## Identifying semantic slots in your designs
 
-Semantic slots should be assigned based on the function of a page element. The name of a semantic slot can quickly tell you how it’s meant to be used. You can find all existing semantic slots and their use case examples in the [Fabric semantic colors documentation](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ISemanticColors.ts).
+Semantic slots should be assigned based on the function of a page element. The name of a semantic slot can quickly tell you how it’s meant to be used. You can find all existing semantic slots and their use case examples in the [Fabric semantic colors documentation](https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots).

```

Comment:
Please remove the locale from the URL... MSDOCS adds this dynamically based on the user's user agent.